docs/spinitron.json

```diff
@@ -753,6 +756,160 @@
           }
         }
       }
+    },
+    "/non-music-items": {
+      "get": {
+        "summary": "Returns non-music items optionally filtered by {start} and/or {end} datetimes",
+        "description": "Get non-music items optionally filtered by a datetime range. Only past non-music items will be returned.\n",
+        "tags": [
+          "Non-music item"
+        ],
+        "parameters": [
+          {
+            "name": "start",
+            "description": "The datetime starting from items must be returned.\n",
```

Comment:
i think this an error in docs for `GET /spins` that you copied to here.

if the query param is not in the request then the server assumes `"now"`.

Reply:
"Return items with timestamps at or after this datetime"